appimage/build_appimage.sh

```diff
@@ -164,7 +166,10 @@ copy_monkey_island_to_appdir() {
   cp "$1"/__init__.py "$INSTALL_DIR"
   cp "$1"/monkey_island.py "$INSTALL_DIR"
   cp -r "$1"/common "$INSTALL_DIR/"
-  cp -r "$1"/monkey_island "$INSTALL_DIR/"
+  if ! timeout "${ISLAND_DIR_COPY_TIMEOUT}" cp -r "$1"/monkey_island "$INSTALL_DIR/"; then
+    log_message "Copying island files takes too long. Maybe you're copying a dev folder instead of a fresh repository?"
+    exit
+  fi
```

Comment:
```suggestion
  if ! timeout "${ISLAND_DIR_COPY_TIMEOUT}" cp -r "$1"/monkey_island "$INSTALL_DIR/"; then
    log_message "Copying island files takes too long. Maybe you're copying a dev folder instead of a fresh repository?"
    exit 1
  fi
```